### PR TITLE
Add simulate details to README.md

### PR DESCRIPTION
**Pull request by dlebu**
@panarasi Can you review these changes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,14 +81,37 @@ In the Command Palette, type ```Cordova``` and choose a command.
 
 ![Cordova commands](images/command-palette.png)
 
-There are only a couple of them for now but that list will grow.
-
 The **Build** command triggers ```cordova build``` and builds for all platforms that you've added to the project.
```
Comment (MSLaguana) [inline README.md]: Should this be a single-backtick quote like the other ones you've added below?

Comment (MSLaguana) [inline README.md]: Just checked the output and it looks fine, so feel free to ignore this.

**Comment by MSLaguana** [comment]
Looks good to me.

**Comment by dlebu** [comment]
Thanks everyone. Merging to master.

**Pull request merged**